### PR TITLE
Japanese localization

### PR DESCRIPTION
**Pull request by smorimoto**
Current progress:

- [x] `intro.yml`
- [x] `proposals.yml`
- [x] `site.yml`
- [x] `stage3.yml`

### CODE REVIEW

File: _data/ja/intro.yml
Comment (rkirsling) [inline _data/ja/intro.yml]: TC39自体のことなので、こんな感じですかね。
```suggestion
  ecma-logo: <a class="ecma-logo-box" href="https://www.ecma-international.org/"> <img src="assets/img/ecma-logo.svg" alt="Ecma International" /> </a> <br/>所属の委員会です。
```

**Review by rkirsling — COMMENTED** [review]
Thanks for doing this!! :bow:

File: _data/ja/intro.yml
Comment (smorimoto) [reply _data/ja/intro.yml]: 確かにこちらのほうがより自然に聞こえると思います！

**Comment by smorimoto** [comment]
Note: @sosukesuzuki and @Naturalclar will help me with this PR so they will add some random commits to my fork repository, but that is intended.

**Comment by sosukesuzuki** [comment]
@Naturalclar Are you already working on `stage3.yml`?

**Comment by Naturalclar** [comment]
@sosukesuzuki Not at the moment, feel free to take it :+1:

**Comment by codehag** [comment]
Amazing, thank you. I am unable to ok the content, can someone send me a ping when this has the all clear? I don't know if everyone has the right privileges. Let  me know if anything is needed.

**Comment by smorimoto** [comment]
@rkirsling may be able to review Japanese content. If privilege means the privilege to merge this PR, then probably no one here has it except you. Of course, I will ping you when we make sure that we can merge.

**Review by rkirsling — COMMENTED** [review]
Yeah, I'm happy to review, I just evidently don't have write access.
Looks like there's one more page to be localized, so I just have one more comment for now.

File: _data/ja/proposals.yml
Comment (smorimoto) [inline _data/ja/proposals.yml]: After reviewing it again, we may need to fix this to keep the translation consistent.

CC: @Naturalclar 

```suggestion
last-presented-alt: 直近のプロポーザルのノート
```

File: _data/ja/site.yml
Comment (smorimoto) [reply _data/ja/site.yml:16]: 例えば、"コントリビュート"以外でも、"提案"を"プロポーザル"と訳しているのですが、これは日本人が日本人であるにも関わらず、日本語ではなく英語を部分的に好んで使う傾向からきています。日常的な例では、"テレビ"よりも"受像機"が全く使われていないのもこのためです。つまり、どちらも完全に正しいのですが、より日常的な会話の中で自然とされるのは"コントリビュート"であるということです。

For example, besides "コントリビュート", "proposal" is translated as "プロポーザル", which stems from the fact that even though Japanese people are Japanese, they prefer to use English partially, not Japanese. This is why we don't use "受像機" at all in our daily examples. (​Probably many people just call it TV.) In other words, both are entirely correct, but the more natural part of everyday conversation is "コントリビュート". (I don't know if I can explain well in English, But your Japanese skills are better than my English skills, so I'm not very worried about it 🙄)

Comment (rkirsling) [reply _data/ja/site.yml:16]: Aw, your English is great! 動詞だからか（？）、何となく確認しときたいと思っただけです 😅

Comment (smorimoto) [reply _data/ja/site.yml:16]: ありがとう 😄 私も日本語の専門家ではないので細かな使い分けに関するルールのようなものは正確には分かりませんが、恐らく @yaminoma はこの分野に詳しいです。

File: _data/ja/proposals.yml
Comment (rkirsling) [reply _data/ja/proposals.yml]: 僕もコメントしようかと思ったんですけど、「プロポーザルを提案」という風に使っているみたいですね。どちらにしても、大事なのは「プロポーザル」と「その発表」を使い分けることだと思います。

Comment (smorimoto) [reply _data/ja/proposals.yml]: では、この部分は"提案"を"プロポーザル"に置き換えようかと思います。(「その発表」は恐らく「提案」を意図していますよね？)

Comment (rkirsling) [reply _data/ja/proposals.yml]: （たとえば「この**プロポーザル**は一年も**発表**されず、ずっと同じ段階に留まっている」という意味で。）

Comment (smorimoto) [reply _data/ja/proposals.yml]: ​Was "発表" used somewhere? (Maybe I'm misunderstanding something.) You used the word "段階", So is it related to the case when the stage is changed?

Comment (rkirsling) [reply _data/ja/proposals.yml]: Sorry for the confusion. 😅 This PR doesn't currently use 発表 (and it's okay if we use a different word), but I mean that these notes are for each **presentation** about the proposal in a meeting, either to request stage advancement or to give a simple status update.

Comment (smorimoto) [reply _data/ja/proposals.yml]: Oops! I thought the word before translation was "proposal"... I'm so sorry to complicate the discussion because of my misunderstanding!

**Comment by smorimoto** [comment]
For `stage3.yml`, I think it would be better to translate only the description section. What do you think? For example, "Top-level await" will lose its meaning when translated.

**Comment by smorimoto** [comment]
I rebased to the latest master.

**Comment by smorimoto** [comment]
@codehag By the way, the URLs for `.at()` and `Import Assertions` are wrong and seem to be broken.

**Comment by smorimoto** [comment]
@sosukesuzuki Would you still like to tackle this? (This isn't a huge rush btw, but just want to check.)

**Comment by sosukesuzuki** [comment]
Yes. Sorry for delay. I'll create PR today.

**Comment by smorimoto** [comment]
All right, thanks! No need to apologize :) (Consider your contributions to us as much appreciated, but certainly not demands. Guard your own time carefully.)

File: _data/ja/proposals.yml
Comment (berlysia) [inline _data/ja/proposals.yml]:
```suggestion
specification-alt: 仕様文を読む
```

Comment (smorimoto) [reply _data/ja/proposals.yml]: Good catch!

Comment (smorimoto) [reply _data/ja/proposals.yml]: Thanks @berlysia!

File: _data/ja/proposals.yml
Comment (berlysia) [inline _data/ja/proposals.yml]: Is this forgotten, or was it intentional?

Comment (smorimoto) [reply _data/ja/proposals.yml]: I completely missed it, so I just pushed commit to fix it. You just beat me!

**Comment by smorimoto** [comment]
I think this is already in a mergeable state. The final quality of this is very good, and if no minor problems are found, we can merge this.

**Review by rkirsling — APPROVED** [review]
Looks good aside from a few last comments! 🙂

File: _data/ja/stage3.yml
Comment (smorimoto) [reply _data/ja/stage3.yml]: 全体的にもう少し改善してみました、どうでしょう？

Comment (rkirsling) [reply _data/ja/stage3.yml]: LGTM :+1:

**Review by rkirsling — APPROVED** [review]

**Comment by smorimoto** [comment]
Thanks to everyone who helped me with this PR. I probably couldn't have done such a high-quality translation without your help. (Actually, I'm not very good at Japanese.) And @codehag, this is finally really mergeable. Please merge this when you have time.

**Comment by codehag** [comment]
Thank you everyone!

**Pull request merged**
